src/test/java/com/nike/cerberus/service/JwtServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.nike.cerberus.service;
+
+import com.nike.cerberus.jwt.CerberusJwtClaims;
+import com.nike.cerberus.jwt.CerberusJwtKeySpec;
+import com.nike.cerberus.jwt.CerberusSigningKeyResolver;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.JwsHeader;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.*;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+public class JwtServiceTest {
```

Comment:
I would like to see more tests here.

Specifically a test proving that no-alg as the alg doesn't circumvent auth.